src/libraries/Microsoft.PowerFx.Core/Public/Config/PowerFxConfig.cs

```diff
@@ -28,9 +28,12 @@ public sealed class PowerFxConfig
         {
             DebugName = "DefaultConfig"
         };
-                
+
+        [Obsolete("Use Config.EnumStore or symboltable directly")]
         internal EnumStoreBuilder EnumStoreBuilder => SymbolTable.EnumStoreBuilder;
 

```

Comment:
We should remove this. 
Can't remove quite yet  - it's used in DataverseIntellisense. DataverseIntellisenseData (although not when we have composed symbol tables, so they don't hit the bug) 

PowerApps doesn't use it.